test/Rakefile.rb

```diff
@@ -12,6 +12,13 @@
   t.warning = true
 end
 
+desc 'Run test_unit based plugin tests'
+Rake::TestTask.new(:plugintest) do |t|
+  t.test_files = Dir["#{ENV['BASE_DIR']}/test/**/*_plugintest.rb"].sort
+  t.verbose = true
+  #t.warning = true
+end
+
 desc 'Run test_unit based system tests'
```

Comment:
require 'fluent/test' in the in_sudo_tailplugintest.rb resulted in pages of warnings thrown by fluentd from their code itself so I made this new test task